src/components/MapComponent.tsx

```
@@ -1,55 +1,153 @@
+            const effectLocation = location || contactConfig.location.venueName;
+            
+            try {
+                // Dynamically import Leaflet at runtime
+                const L = await import('leaflet').then(module => module.default);
+                
+                // No need to dynamically load CSS as we'll import it directly
+                
+                // Fix missing marker icon issue by manually setting the icon path to the CDN
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                const IconDefault = L.Icon.Default as any;
+                delete IconDefault.prototype._getIconUrl;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)